xcube_sh/sentinelhub.py

```diff
@@ -90,6 +139,11 @@ def dataset_names(self) -> List[str]:
         obj = json.loads(resp.content)
         return obj.get('data')
 
+    # def refresh_session(self):
+    #     client = oauthlib.oauth2.BackendApplicationClient(client_id=self.client_id)
+    #     # self.session = requests_oauthlib.OAuth2Session(client=client, token=self.token)
+    #     self.session = SentinelOAuth2Session(client=client, token=self.token)
+
```

Comment:
Comment, why this is commented out, reason why this should stay here. Or remove, if there is no use.